Prevent curl-config from telling curl's dependents that they have to link with
all of curl's dependencies as well.
--- configure.orig	2023-09-13 08:41:45
+++ configure	2023-09-13 08:42:48
@@ -14477,7 +14477,7 @@
   else
     whole_archive_flag_spec=''
   fi
-  link_all_deplibs=yes
+  link_all_deplibs=no
   allow_undefined_flag=$_lt_dar_allow_undefined
   case $cc_basename in
      ifort*|nagfor*) _lt_dar_can_shared=yes ;;
